pkg/providers/cloudstack/testdata/cloudstack_config_multiple_instances.ini

```diff
@@ -0,0 +1,11 @@
+verify-ssl = false
```

Comment:
Looks like this change would break CAPC actually. They're expecting everything under [Global]. https://github.com/kubernetes-sigs/cluster-api-provider-cloudstack/blob/main/pkg/cloud/client.go#L56-L62. For backwards compatibility, can we keep the option of the global tag, and then add additional instances below it?

Reply:
If we want to keep the backward compatibility we should keep [Global] section as the first and mandatory failure domain and any additional ones should go under the global section. However, the old versions of capc would use only the global section as a single and only ACS management endpoint. A newer version of capc will treat each section as a failure domain. In this case, `verify-ssl` property can be shown in all sections and the values can be different each other. If we take this route creating a separate cmk config file per failure domain should be taken because `verify-ssl` is a global flag in a cmk config file.

Reply:
I don't see any issues with having multiple cmk config files. That feels like an implementation detail that shouldn't affect the customer. It also gives them the flexibility to determine if they want to verify ssl for different failure domains independently.